library/alpine

```diff
@@ -14,10 +14,10 @@ s390x-Directory: s390x/
 i386-Directory: x86/
 amd64-Directory: x86_64/
 
-Tags: 3.14.0, 3.14, 3, latest
+Tags: 3.14.1, 3.14, 3, latest
 Architectures: arm64v8, arm32v6, arm32v7, ppc64le, s390x, i386, amd64
 GitFetch: refs/heads/v3.14
-GitCommit: 6cde5b1c2838b647382fdbc71b5d3cef54c5c157
+GitCommit: 571b69f12f4a3891f48b308ed9611e018cc862f7
```

Comment:
The commit seems to be missing (https://github.com/alpinelinux/docker-alpine/commit/571b69f12f4a3891f48b308ed9611e018cc862f7). Maybe it needs a force push for the `v3.14` branch?

Reply:
Indeed. sorry.

fixed now.